index.html

```diff
@@ -3417,7 +3417,7 @@ <h2><span class="chunk">iCCP</span> Embedded ICC profile</h2>
           </table>
 
           <p>The profile name may be any convenient name for referring to the profile. It is case-sensitive. Profile names shall
-          contain only printable Latin-1 characters and spaces (only character codes 32-126 and 161-255 decimal are allowed).
+          contain only printable Latin-1 characters and spaces (only code points 0x20-7E and 0xA1-FF are allowed).
```

Comment:
Nit: I would probably use the Unicode `U+xxxx` notation for code points here and on line 3927, but I'm okay with this.

Reply:
The text `0x20-7E and 0xA1-FF` is your suggestion, actually :)